Anagrama.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+#include <string.h>
+
```

Comment:
This program tests whether a word or phrase is an anagram of another one.
Sample Input:
Thats one small step for a man. One giant leap for mankind. Neil Armstrong
Thin man ran... makes a large stride... left planet... pins flag on moon... on to Mars!
Sample Output:
True